test/avocado/selenium-base.py

```diff
@@ -7,27 +7,31 @@
 if not machine_test_dir in sys.path:
     sys.path.insert(1, machine_test_dir)
 
-from avocado import main
-from avocado.utils import process
-from seleniumlib import SeleniumTest, user, clickable, passwd
+from seleniumlib import SeleniumTest, user, clickable, passwd, visible
 
 class BasicTestSuite(SeleniumTest):
     """
     :avocado: enable
     """
     def test10Base(self):
-        out = process.run("hostname", shell=True)
+        self.wait_id('server-name')
+        self.error = False
+
+    def test15BaseSSHKeyAdded(self):
+        self.login()
+        self.logout()
+        out = self.machine.execute("hostname")
         server_element = self.wait_id('server-name')
-        self.assertTrue(str(out.stdout.decode("utf-8"))[:-1] in str(server_element.text))
-        self.error=False
+        self.assertTrue(out.strip() in str(server_element.text))
```

Comment:
Just a hint for the future: Use `self.assertIn(out.strip(), str(server_element.text))` for that. If the test fails, you will then get the actual strings printed out, which is much more helpful for debugging.